plotly_calplot/date_extractors.py

```diff
@@ -13,26 +13,16 @@ def get_date_coordinates(
 ) -> Tuple[Any, List[float], List[int]]:
     month_days = []
     for m in data[x].dt.month.unique():
-        month_days.append(data.loc[data[x].dt.month == m].max()[x].day)
+        month_days.append(data.loc[data[x].dt.month == m, x].max().day)
```

Comment:
FYI: this fixes a pandas `.agg` FutureWarning:
```
[...]/plotly_calplot/date_extractors.py:16: FutureWarning:

Dropping of nuisance columns in DataFrame reductions (with 'numeric_only=None') is deprecated; in a future version this will raise TypeError.  Select only valid columns before calling the reduction.
```